devtools/conda-envs/test_env.yaml

```diff
@@ -2,6 +2,7 @@ name: paprika-dev
 channels:
   - conda-forge
   - omnia
+  - mwt
```

Comment:
Any thought about getting `intermol` onto `conda-forge`? I can't even find this channel after spending five minutes of Googling.

Reply:
Yeah, that's just my "personal" channel where I dump stuff from time to time. It's not really intended to be used by others, although I didn't put anything in place to prevent that. I don't believe I advertised this build, but again there's nothing hidden about it.

Putting it on `conda-forge` is something that probably should be done at some point. It should be feasible - the deps are really light - but some of the package structure might need a refresh. I can take it on.

Reply:
Ah! I didn't even connect `mwt` with @mattwthompson. Ha.

Reply:
Yeah, I found the package by searching `Intermol` on the anaconda website. It's not urgent, but if you can get it to `conda-forge` then :heart_eyes: